test/built-ins/BigInt/asIntN/arithmetic.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: BigInt.asIntN arithmetic test cases
+info: >
+  BigInt.asIntN ( bits, bigint )
+
+  3. Let mod be a BigInt representing bigint modulo 2**bits.
+  4. If mod ≥ 2**bits - 1, return mod - 2**bits; otherwise, return mod.
```

Comment:
The `features.txt` mentions `Symbol` as a "standard language feature" that's "largely maintained for historical reasons". How important is it to declare my dependency on that feature?

And for the `arrow-function` issue below, is there any value in me avoiding arrow functions to avoid depending on arrow functions? Is there any concern for allowing a partially-conformant implementation to test their support for BigInt without also needing support for arrow functions? If not, I can easily just throw `arrow-function` into the features declaration.